index.d.ts

```diff
@@ -0,0 +1,27 @@
+export = launchWebAuthFlow;
+
+declare function launchWebAuthFlow(options: WebAuthFlowOptions): ResponseUrl;
+
+interface WebAuthFlowOptions {
+  url: string,
+  redirect_uri: string,
+  interactive?: boolean,
+  alwaysUseTab?: boolean,
+  windowOptions?: {
```

Comment:
Is it possible to reuse type from other sources? Is there an official/3rd-party webextensions API type declaration that can be used?

Reply:
Mozilla has an official auto-generated type declaration https://www.npmjs.com/package/@types/webextension-polyfill
however, I'm not sure how to create a type based on another type declaration, I'll investigate.